package.json

```diff
@@ -2,21 +2,24 @@
   "name": "@side/renovate-config",
   "description": "Side Inc. shared renovate config templates",
   "scripts": {
-    "prepare": "husky",
+    "postinstall": "husky",
+    "prepack": "pinst --disable",
+    "postpack": "pinst --enable",
     "format": "prettier --write .",
     "format:check": "prettier --check .",
     "lint": "eslint . --ext .json,.js --max-warnings 0"
   },
   "devDependencies": {
-    "@side/eslint-config-base": "2.1.0",
-    "@side/lint-staged-config": "1.0.0",
-    "@side/prettier-config": "1.0.0",
-    "eslint": "^8.56.0",
+    "@side/eslint-config-base": "2.2.0",
+    "@side/lint-staged-config": "1.1.0",
+    "@side/prettier-config": "1.1.0",
+    "eslint": "^8.57.1",
```

Comment:
Should we pin this?

Reply:
Is there a strong reason why we want to pin all our dev dependencies? I didn't change this here but I can pin it if we really want to.